### PR TITLE
fix to issue #94

### PR DESCRIPTION
**Pull request by k-for-code**
Fixes # https://github.com/Layout-Parser/layout-parser/issues/94#issue-1040387217
#94 
The issue was, all digit sequences were inferred as float, with this fix all text (numeric + non-numeric) will be inferred as string and the user can change it to their desired data type.
But with this fix, the user will be required to change the numeric data type columns.
i could not find any better solution other than this.

### CODE REVIEW

File: src/layoutparser/ocr/tesseract_agent.py [hunk 1]
```diff
@@ -89,10 +90,13 @@ def _detect(self, img_content):
...
-        res["data"] = pd.read_csv(
-            io.StringIO(_data), quoting=csv.QUOTE_NONE, encoding="utf-8", sep="\t"
-        )
+        _data = pytesseract.image_to_data(img_content, lang=self.lang, output_type=Output.DICT, **self.configs)
+        _df = pd.DataFrame(_data, dtype=str)
+        _cols = list(_df.columns())
+        _cols.remove('text')
+        for col in _cols:
+            _df[col] = _df[col].astype(int)
+        res['data'] = _df
```
Comment (lolipopshock) [inline src/layoutparser/ocr/tesseract_agent.py:99]: Can you try the following code: 

```
_data = pytesseract.image_to_data(img_content, lang=self.lang, **self.configs)
df = pd.read_csv(
   io.StringIO(_data), quoting=csv.QUOTE_NONE, encoding="utf-8", sep="\t"
)
df['text'] = df['text'].astype('str')
res["data"] = df
```

Comment (k-for-code) [reply src/layoutparser/ocr/tesseract_agent.py:99]: @lolipopshock  sorry it does not, I have tried this
and ya i get it, the for loop and all that stuff looks ugly :)

here's the screenshot 
layout_parse

Comment (lolipopshock) [reply src/layoutparser/ocr/tesseract_agent.py:99]: I see -- it's the issue from floating point numbers `.0` right?

Comment (k-for-code) [reply src/layoutparser/ocr/tesseract_agent.py:99]: Yes

**Comment by lolipopshock** [comment]
I think the new solution can solve your issue -- see example below: 

Let's save we have a csv file 
```
Col_A, Col_B
, 1
2, 3
245.0, 
```
And if we read it via:
```
df = pd.read_csv("test.csv", converters={"Col_A": str})
```
We have 

 Test | B
 -- | --
  | 1
2 | 3
245.0

(There's no `.0` for 2 in the 2nd row and 1st col.

**Pull request merged**